src/neuroconv/basedatainterface.py

```diff
@@ -32,6 +38,7 @@ def get_metadata_schema(self):
             title="Metadata",
             description="Schema for the metadata",
             version="0.1.0",
+            properties=dict(NWBFile=get_schema_for_NWBFile(), Subject=get_schema_from_hdmf_class(Subject)),
```

Comment:
Important enough for a changelog entry? All DataInterfaces now generate NWBFile and Subject metadata schemas